config/config.js

```diff
@@ -27,7 +27,9 @@ module.exports = {
     force_https: process.env.TOOMEAN_APP_FORCE_HTTPS || false,
     // default is 15 minutes
     requireEmailVerification: process.env.TOOMEAN_APP_REQUIRE_EMAIL_VERIFICATION || false,
-    emailVerificationTTL: process.env.TOOMEAN_APP_EMAIL_VERIFICATION_TTL || 15 * 60 * 1000
+    emailVerificationTTL: process.env.TOOMEAN_APP_EMAIL_VERIFICATION_TTL || 15 * 60 * 1000,
+    allowRegistration: process.env.TOOMEAN_APP_ALLOW_REGISTRATION || true,
+    defaultUserRole: process.env.TOOMEAN_APP_DEFAULT_USER_ROLE || 'user'
```

Comment:
Good call, although I'm sure there's hard coded stuff everywhere that changing this would break.